src/app/pages/datasets/components/dataset-form/sections/other-options-section/other-options-section.component.ts

```diff
@@ -212,7 +212,7 @@ export class OtherOptionsSectionComponent implements OnInit, OnChanges {
     }
 
     this.store$.pipe(waitForSystemInfo, untilDestroyed(this)).subscribe((systemInfo) => {
-      if (!systemInfo.license || !systemInfo.license.features.includes(LicenseFeature.Dedup)) {
+      if (!systemInfo.license?.features.includes(LicenseFeature.Dedup)) {
```

Comment:
If license is empty like original condition suggests it can be you'll have:
`undefined.includes(LicenseFeature.Dedup)`